backend/src/WebServer/controllers/mail/MailController.py

```diff
@@ -0,0 +1,47 @@
+from src.util.LogFactory import LogFactory
+from src.WebServer.decorators.HTTPLogger import http_logger
+from src.WebServer.WebServerInit import WebServerInit
+from src.util.ErrorFactory import errorStackTrace
+from src.Mail.MailQ import MailQ
+import re
+
+from flask import Flask, jsonify, request
+
+flask_ref: Flask = WebServerInit.flask
+
+class MailController:
+
+  def __init__(self):
+    LogFactory.MAIN_LOG.info('Start MailController')
+
+  @staticmethod
+  @flask_ref.route('/mail', methods=['POST'])
+  @http_logger
+  def mail_api():
+    try:
+      LogFactory.MAIN_LOG.info("mail api")
+      email = request.args['email']
+      if emailIsValid(email):
+        mailQ = MailQ()
+        redisKey = mailQ.q_size() + 1
+        mailQ.put_item(redisKey, email)
```

Comment:
Can we structure the data we put into the Q as a JSON object? 

It would be a simple JSON object for now which just have something like this:

```
{
"email" : "someemail@blah.com"
}
```